central/clusterinit/backend/backend_test.go

```diff
@@ -401,6 +402,45 @@ func (s *clusterInitBackendTestSuite) TestCRSLifecycle() {
 	}
 }
 
+// Tests auto revocation for a CRS with maxRegistrations = 1.
+
+func (s *clusterInitBackendTestSuite) TestCrsAutoRevocation1() {
```

Comment:
Better to have a meaningful name than a comment that can rot.
```suggestion
func (s *clusterInitBackendTestSuite) TestCrsAutoRevocationOneShot() {
```

Reply:
Done already.